### PR TITLE
add option to toggle tooltips

### PR DESCRIPTION
**Pull request by chrisdrackett**

### CODE REVIEW

File: lib/item-view.js [hunk 1]
```diff
@@ -8,6 +12,7 @@ export default class ItemView {
 
     this.tooltip          = null;
     this.tooltipPlacement = 'auto left';
+    this.showTooltips = atom.config.get(`${CONFIG_PREFIX}showTooltips`);
```
Comment (susisu) [inline lib/item-view.js]: (related to the comment below)
This initial value should also be set by the controller via some new method (as `setTitle()` is used).

File: package.json [hunk 2]
```diff
@@ -53,6 +53,13 @@
       "type": "boolean",
       "default": true,
       "description": "Show Git statuses of files."
+    },
+    "showTooltips": {
+      "order": 4,
+      "type": "boolean",
+      "default": true,
+      "description": "Show tooltips with file path on hover."
+
```
Comment (susisu) [inline package.json]: This empty line should be removed.

File: lib/item-view.js [hunk 3]
```diff
@@ -21,6 +26,18 @@ export default class ItemView {
 
     this.element.appendChild(this.closeButtonElement);
     this.element.appendChild(this.nameElement);
+
+    // event subscriptions
+    this.subscriptions = new CompositeDisposable();
```
Comment (susisu) [inline lib/item-view.js]: I think this event subscription should be in `item-controller.js`.
Can you move this?

**Review by susisu — CHANGES_REQUESTED** [review]
Looks nice! 👍 

I found there are several points to be fixed before merging.
Can you fix them?

Thanks.

File: lib/item-view.js (hunk 1 above)
Comment (chrisdrackett) [reply lib/item-view.js]: done!

File: lib/item-view.js (hunk 3 above)
Comment (chrisdrackett) [reply lib/item-view.js]: done!

**Comment by chrisdrackett** [comment]
@susisu ready for review

File: package.json (hunk 2 above)
Comment (chrisdrackett) [reply package.json]: done!

**Comment by susisu** [comment]
LGTM! (actually the first one seems to be not fixed, but I will fix it later)
Thank you.

**Review by susisu — APPROVED** [review]

**Pull request merged**